pwa-devdocs/src/peregrine/index.md

```diff
@@ -2,9 +2,9 @@
 title: Peregrine
 ---
 
-The Peregrine project is a collection of UI components for Magento PWA projects.
+The Peregrine project is a collection of UI Components for Magento PWA projects.
 Use, extend, or remix these components to create a unique Magento PWA storefront.
 
-Visit the [`peregrine`] repository to contribute to this project.
+Visit the [`Peregrine`] repository to contribute to this project.
```

Comment:
**Peregrine** is the name of the project.
`peregrine` is the name of the repository.

This sentence refers to the repository, so the original `peregrine` form is the correct one.